src/magento-payments/payments.md

```diff
@@ -1,3 +1,58 @@
 ---
 title: Magento Payments
----
\ No newline at end of file
+---
+
+Magento Payments is your turnkey self-service solution, including sandbox testing and a super simple setup, for providing robust and secure payment processing for your Magento websites.
+
+Whether you are a small business, mid-market contender, or large enterprise, our pre-installed payments and risk solution will help you reduce operational overhead while increasing revenue and giving you useful tools to improve the entire shopper experience.
+
+All the tools needed to implement and operate Magento Payments are fully integrated into the Magento Admin---manage everything in one spot with ease.
+
+Magento Payments is...
+
+* Easy to set up and maintain
+* Designed to maximize your profit
+* Safe and secure
+* Fully-featured to meet all your needs
+* Self-contained within the Magento Admin
+
+IMAGE
+
+## Features
+
+{:.bs-callout-info}
+Some of the features mentioned here may not yet be available for the Early Adopter Program (EAP).
+
+Magento Payments is your one-stop-shop for online checkout (from settlement and refunds to getting paid!) We make setup and configuration painless and we provide powerful tools to give you the insight and control you need to create the best experience for your buyers.
```

Comment:
As a general rule, exclamations are a little over the top in the tech docs. Marketing and tweets can get away with it a bit more.

Reply:
I understand what you are saying. My hope was that, given our non-technical audience, we could inject a bit more personality into the documentation to instill some confidence in this new product.